daml/Tests/Distribution/Syndication/Setup.daml

```diff
@@ -123,60 +123,62 @@ setup = do
   aiCid1 <- openAccount operator bondRegistrar bondRegistrar $ createLabel bondRegistrar bondRegistrar
   aiCid2 <- openAccount operator bondRegistrar issuer $ createLabel bondRegistrar issuer
   aiCid3 <- openAccount operator bondRegistrar custodian1 $ createLabel bondRegistrar custodian1
+  aiCid3a <- openAccount operator bondRegistrar custodian1 $ "Segregated-" <> partyToText investor1 <> "@" <> partyToText custodian1 <> "@" <> partyToText bondRegistrar
```

Comment:
For investor1 we have this segregated account under custodian and I can see that it is set as a security account instead of omnibus account.
I guess I am confused about what account should be put as a omnibus account and what account should be insert into map of security/cash accounts.
@georg-da do you mind to clarify it for me, thanks!

Reply:
This is simply an example here that you _can_ use a segregated account for any given party, if you have to. Generally, the choice of whether to use a segregated or omnibus account depends on your operating model / regulatory / compliance constraints.